docs/conf.py

```diff
@@ -19,7 +19,7 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('../'))
```

Comment:
On 2016-02-29 16:43, Aldo Hoeben wrote:
> In docs/conf.py [1]:
>
>> @@ -19,7 +19,7 @@
>> # If extensions (or modules to document with autodoc) are in another
> directory,
>> # add these directories to sys.path here. If the directory is
> relative to the
>> # documentation root, use os.path.abspath to make it absolute, like
> shown here.
>> -#sys.path.insert(0, os.path.abspath(.))
>> +sys.path.insert(0, os.path.abspath(../))
>
> Why do we need to include the parent dir? What fixes this?
>
> --
> Reply to this email directly or view it on GitHub [2].
>
> 	*

otherwise you need to install Pyre first to generate the documentation.